README.md

```diff
@@ -1 +1,16 @@
 # Apple Clone
+
+Clone of the Apple landing page from year 2014.
+
+##### Built With
++ HTML5
++ CSS3
+
+# Cloned Page URL
+
+URL:  [Apple (2014)](https://web.archive.org/web/20140401014630/http://www.apple.com/)
+
+# Authors
++ Rohan Sundar [https://www.github.com/rsundar](https://www.github.com/rsundar)
+
+Live Version:  [https://htmlpreview.github.io/?https://github.com/rsundar/Apple-clone/blob/dev-branch/index.html](live preview)
```

Comment:
#### `HINT`

Add a descriptive readme. Details like assignment link (from Odin), screenshot and what was learned during the project can make the project more descriptive.

Please, [style markdown](https://guides.github.com/features/mastering-markdown/) files appropriately to make the readme look more professional as this is the main file that will be displayed in the repo homepage.